content/english/security/1- Meeting Alice, Bob and Eve/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Meeting Alice, Bob and Eve"
+description: ""
+draft: false
+weight: 1
+translationKey: "security-1"
+---
+
+![Cannot load image](Bob.png?classes=border,shadow)
```

Comment:
I'll wait for you to merge with Nuevo/Hack2020 so that we see these as changes to existing files rather than new file additions